### PR TITLE
chore: edited links

### PR DESCRIPTION
**Pull request by Chakravarthi0**

### CODE REVIEW

**Review by talrejah02 — APPROVED** [review]
good to go.

File: pages/authentication/signin.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h2 class="nav-logo">WitTest</h2>
             <input class="input nav-search" type="text" placeholder="Search" />
```
Comment (AbhishekkGautam) [inline pages/authentication/signin.html:32]: as an alternative, we can also give type="search" for search input. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search

**Review by AbhishekkGautam — APPROVED** [review]
LGTM

File: pages/authentication/signin.html (hunk 1 above)
Comment (Chakravarthi0) [reply pages/authentication/signin.html:32]: Thanks for the suggestion, I will check it out.

**Pull request merged**